docs/index.html

```diff
@@ -33,6 +33,8 @@
             overflow: hidden;
             margin: 0 !important;
             padding: 0 !important;
+            height: 100%;
+            width: 100%;
```

Comment:
This is an important (breaking!) change that should be documented in `eframe/CHANGELOG.md`